api/shapes/settings.ts

```diff
@@ -102,6 +102,9 @@ export interface Settings {
   /** The initial status of assume masterwork in the loadout optimizer. */
   readonly loAssumeMasterwork: boolean;
 
+  /** The optimizers material spend tier, effects armors maximum energy when calcuating sets. */
+  readonly loUpgradeSpendTier: number;
```

Comment:
Should this be an enum?

Reply:
Yes, I didn't know we could use enums for api types

Reply:
Done